docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -3,6 +3,15 @@ LABEL maintainer="Hugging Face"
 
 ARG DEBIAN_FRONTEND=noninteractive
 
+# The following `ARG` are mainly used to specify the versions explicitly & directly in this docker file, and not meant
+# to be used as arguments for docker build (so far).
+
+ARG PYTORCH='1.11.0'
+# (not always a valid torch version)
+ARG INTEL_TORCH_EXT = '1.11.0'
+# Example: `cu102`, `cu113`, etc.
+ARG CUDA='cu113'
```

Comment:
In other dockerfiles, if this is not specified, we default to the latest version. This means that we don't have to actively update the values here. Would it work here as well?

Reply:
I think you are talking about [transformers-pytorch-gpu](https://github.com/huggingface/transformers/blob/main/docker/transformers-pytorch-gpu/Dockerfile).

Here we actually don't want to to use the latest version (implicitly). The main motivation is that we should use new PyTorch release after validating the 3rd party libraires are ready.

Here is @stas00 comment
https://github.com/huggingface/transformers/pull/17586#issuecomment-1148939968

Therefore `transformers-pytorch-gpu` might need to be changed too.

- So far we don't actually use those ARG yet I think. In the future, if we need to build the latest versions, as well as versions that works with the 3rd libraries, we need to figure out a better/clean/flexible solution.

Reply:
Ok, then sounds good to me!